subsys/bluetooth/controller/ll_sw/nordic/lll/lll_vendor.h

```diff
@@ -27,5 +27,7 @@
 #define EVENT_RX_TO_US(phy) ((((((phy)&0x03) + 4)<<3)/BIT((((phy)&0x3)>>1))) + \
 				  EVENT_RX_JITTER_US(phy))
 
-/* TODO - fix up numbers re. HW */
-#define EVENT_RX_TX_TURNAROUND(phy)  ((phy) == 1?100:((phy) == 2 ? 80:900))
+/* Using conservative turnaround time, considering slowest CPU timings that is
+ * required to meet tIFS of 150 us.
+ */
+#define EVENT_RX_TX_TURNAROUND(phy)   150
```

Comment:
How come this isn't related to the `phy` value anymore? The old maximum value was seemingly 900, so relative to that, this isn't very conservative.